sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/api/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.netstandard2.0.cs

```diff
@@ -12,7 +12,9 @@ public partial class AvroSerializationException : System.Exception
     }
     public partial class SchemaRegistryAvroSerializer
     {
-        public SchemaRegistryAvroSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client, string groupName, Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.SchemaRegistryAvroSerializerOptions options = null) { }
+        public SchemaRegistryAvroSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client) { }
+        public SchemaRegistryAvroSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client, string groupName) { }
+        public SchemaRegistryAvroSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client, string groupName, Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.SchemaRegistryAvroSerializerOptions options) { }
```

Comment:
groupName is part of the options in JS because it is optional. It is only needed when serializing but it is valid to use the serializer for deserializing only.

Reply:
I don't think it would make sense to put in the options for .NET, as it makes the mainline scenario more complicated.

Reply:
when serializing, I'd need to do:
```c#
var serializer = new SchemaRegistryAvroSerializer(client, new SchemaRegistryAvroSerializerOptions { GroupName = "groupName"});
```
rather than:
```c# 
var serializer = new SchemaRegistryAvroSerializer(client, "groupName");
```